### ISSUE TITLE
Error with game data when attempting sortie.

### ISSUE THREAD

**Issue by kannasama** [KC3Kai/KC3Kai#2724]
### Versions:
- Chrome: 70.0.3538.110
- OS: Windows 10 (1803)
- 64-bit
- KC3 Kai: 33.5.1 Webstore Build Direct Playmode

### Steps to reproduce:
Simply going to sortie into a regular map results in the screenshot above.

### Actual behaviour:
Error reported with game data, indicating possible API issue.

### Expected behaviour:
No error to be present.

### Screenshots (if applicable):
![image](https://user-images.githubusercontent.com/8433185/49700210-a3be4280-fba9-11e8-94d0-6334d0ab51b1.png)

### Error Report (if applicable):
[KC3-Error-1544374343.txt](https://github.com/KC3Kai/KC3Kai/files/2660768/KC3-Error-1544374343.txt)

**Comment by sinsinpub** [comment]
😮 show me ur local mapinfo data:
Open chrome console (by pressing F12 or menu) from any Strategy Room page, execute this:
```
localStorage.maps
```
paste the output here, lemme check what is missing...

**Comment by kannasama** [comment]
[mapinfo.txt](https://github.com/KC3Kai/KC3Kai/files/2661473/mapinfo.txt)
Attached.

It's worth pointing out that I play across several devices, and the error only came up on one of them.  I've pretty much cleared out all historical data I could, short of completely rebuilding the profile (wanting to avoid having to re-apply all the settings).

**Comment by sinsinpub** [comment]
okay, ur data look mess a bit. before our fix released next time, try this workaround:
execute these codes in the same console mentioned above:
```
var maps = KC3SortieManager.getAllMapData();
maps["m71"].kills = false; maps["m72"].kills = false;
delete maps["m00"]; delete maps["m-10"];
KC3SortieManager.setAllMapData(maps);
```

**Comment by kannasama** [comment]
I haven't used that laptop for a while.  Just tried the workaround, and the error's resolved.

**Comment by mikenzb** [comment]
Fixed in release 33.5.2
